data/entities.yaml

```diff
@@ -611,7 +621,7 @@
     of the original entity.
   - For entities which do not grow, 'harvest' behaves exactly the same
     as 'grab'.
-  capabilities: [grab, harvest]
+  capabilities: [grab, harvest, give, place, install]
```

Comment:
@byorgey was this intended? I thought it was a strict upgrade so that I could use a harvester instead of a grabber.

Reply:
This was definitely intended.  It is not supposed to be a strict upgrade: you start with no harvesters but they are supposed to be easy to build (you can make one with three trees); whereas you start with a bunch of grabbers but they are more difficult to build.  You can use a harvester instead of a grabber to do `grab`, but not to do the other things a grabber can do.

Reply:
I would at least add **place**, so it can be used to grab&fetch without salvaging. :thinking:

That sounds realistic to me and leaves **give** and **install** to the grabber.

Reply:
OK, sure, that sounds good to me.